qa/SidechainTestFramework/sc_test_framework.py

```diff
@@ -190,6 +205,8 @@ def main(self):
                           help="Stores jacoco flag assigned to current test")
         parser.add_option("--allforks", dest="all_forks", default=False, action="store_true",
                           help="Enables all existing fork points at the first epoch")
+        parser.add_option("--maxhistoryrewritinglength", dest="maxhistoryrewritinglength", type=int, default=100, action="store",
```

Comment:
Please, remember to update README file with the new command line option

Reply:
Done